pkg/controllers/install_reconcile_test.go

```diff
@@ -101,10 +103,14 @@ var _ = Describe("k8s/install_reconcile_test", func() {
 		// Reset the pod runner output to dump the compat21 node number
 		fpr.Results = cmds.CmdResults{
 			names.GenPodName(vdb, sc, 1): []cmds.CmdResult{
-				{}, // Check for stale admintools.conf
+				{}, // Remove old admintools.conf
```

Comment:
could you explain these changes to me?

Reply:
the next 2 lines included

Reply:
I added new debug code to capture the state of admintools.conf.  This adds extra calls to the pod runner.  There were two testcases that were sensitive to this and had to be updated.